source/Playnite.DesktopApp/Themes/Desktop/Default/DefaultControls/CheckBox.xaml

```diff
@@ -17,8 +17,9 @@
                 <ControlTemplate TargetType="{x:Type CheckBox}">
                     <Border Background="{TemplateBinding Background}"
                             Padding="{TemplateBinding Padding}">
-                        <StackPanel Orientation="Horizontal">
+                        <DockPanel>
                             <Border x:Name="BulletBorder" CornerRadius="{DynamicResource ControlCornerRadius}"

```

Comment:
Children elements should have DockPanel.Dock set to specific values so it's clear how they are docked.

Reply:
Done